Makefile

```
@@ -69,7 +69,8 @@ NEW_URLS=$(shell scripts/ignore_new_urls.sh 2> /dev/null)
 URL_IGNORE:=$(URL_IGNORE)$(NEW_URLS)
 
 ## validate-site: Builds the site and validates the pages. This is used for CI
+CONFIG='{\"enforce_https\": false,\"check_internal_hash\": false, \"check_external_hash\":false,\"ignore_empty_alt\":true}'
 .PHONY: validate-site
 validate-site: build-hugo
-	${DORP} run -t -i --rm -v "$(shell pwd)":/site:z -w /site ${KIALI_HUGO_IMAGE} /bin/bash -c "npm prune && hugo && htmlproofer --assume-extension --check-external-hash --empty_alt_ignore --url-ignore \"${URL_IGNORE}\" ./public"
+	${DORP} run -t -i --rm -v "$(shell pwd)":/site:z -w /site ${KIALI_HUGO_IMAGE} /bin/bash -c "npm prune && hugo && htmlproofer ./public --assume_extension --allow_hash_href true --allow_missing_href true ignore_empty_alt true --ignore_missing_alt true --check_external_hash false --check_internal_hash false --enforce_https false --ignore_urls \"${URL_IGNORE}\""
```

It looks like you are missing `--` for the `ignore_empty_alt` option - all the other options have the `--` prefix except for that one.

```suggestion
	${DORP} run -t -i --rm -v "$(shell pwd)":/site:z -w /site ${KIALI_HUGO_IMAGE} /bin/bash -c "npm prune && hugo && htmlproofer ./public --assume_extension --allow_hash_href true --allow_missing_href true --ignore_empty_alt true --ignore_missing_alt true --check_external_hash false --check_internal_hash false --enforce_https false --ignore_urls \"${URL_IGNORE}\""
```